docs/icd.md

```diff
@@ -1 +1,102 @@
-//todo
+# Interface Control Document (ICD) - `svc-storage`
+
+<center>
+
+<img src="https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png" style="height:250px" />
+
+</center>
+
+## Overview
+
+This document defines the gRPC interfaces unique to the `svc-storage` microservice.
+
+Attribute | Description
+--- | ---
+Status | Draft
+
+## Related Documents
+
+Document | Description
+--- | ---
+[Requirements - `svc-storage`](https://docs.google.com/spreadsheets/d/1OQZGOxQh7gWo3BeZzwwHNSdY84Ghucx2MVADVlXV4AY/edit?usp=sharing) | Requirements for this module
+[Software Design Document (SDD)](./sdd.md) | 
+
+## Frameworks
+
+See the Services ICD.
+
+## gRPC
+
+### Files
+
+These interfaces are defined in protocol buffer files:
+ * [`svc-storage-grpc.proto`](../proto/svc-storage-grpc.proto)
+ * [`svc-storage-grpc-flight_plan.proto`](../proto/svc-storage-grpc-flight_plan.proto)
+ * [`svc-storage-grpc-pilot.proto`](../proto/svc-storage-grpc-pilot.proto)
+ * [`svc-storage-grpc-vehicle.proto`](../proto/svc-storage-grpc-vehicle.proto)
+ * [`svc-storage-grpc-vertipad.proto`](../proto/svc-storage-grpc-vertipad.proto)
+ * [`svc-storage-grpc-vertiport.proto`](../proto/svc-storage-grpc-vertiport.proto)
+
+### Integrated Authentication & Encryption
+
+See Services ICD.
+
+### gRPC Server Methods ("Services")
```

Comment:
Please add a note that says a more detailed API is planned for next release

Reply:
Fixed in https://github.com/Arrow-air/svc-storage/pull/8/commits/fd8adfc5cf128bc1c4bf3081184147d5b5f76c30